src/cmd/cli/command/commands.go

```diff
@@ -637,6 +647,10 @@ func printEndpoints(serviceInfos []*v1.ServiceInfo) {
 	}
 }
 
+func isCommandUsingDefangPlayground(cmd *cobra.Command) bool {
+	return *(cmd.Flag("provider").Value.(*cliClient.Provider)) == cliClient.ProviderDefang
```

Comment:
the default value of `provider` is auto, which checks for the presence of any AWS_* env vars, without which it'll use the playground. The message didn't get printed in this default case.

Reply:
Check https://github.com/defang-io/defang/blob/7aaf30b101c757745ba6991c4b1a6f43def316ef/src/cmd/cli/command/commands.go#L258

We should probably reuse that global var `provider`

Reply:
Playground deployment 
![Screenshot 2024-04-16 at 9 08 04 AM](https://github.com/defang-io/defang/assets/7478236/f7fe2627-40d7-4441-bbe9-95ccb00c7edd)